certified-connectors/Zippydoc/apiDefinition.swagger.json

```diff
@@ -0,0 +1,258 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ZIPPYDOC",
+    "description": "ZIPPYDOC is a platform that allows to merge, transform, filter, extract, report and analyze structured and table based data and to flexibly create data flows",
+    "contact": {
+      "name": "ZIPPYDOC support",
+      "url": "https://support.zippydoc.de",
+      "email": "support@zippydoc.eu"
+    },
+    "version": "1.0"
+  },
+  "host": "api.prod.zippydoc.net",
+  "basePath": "/api/core",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/power-automate": {
+      "x-ms-notification-content": {
+        "description": "OnTableChangeResponse",
+        "schema": {
+          "type": "object",
+          "properties": {
+            "tableId": {
+              "type": "integer",
+              "format": "int32",
+              "description": "ID of the table in ZIPPYDOC system",
+              "title": "tableId"
+            }
+          }
+        }
+      },
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "operationId": "OnTableChange",
+        "summary": "On table changed",
+        "description": "Triggers flow when any table is modified.",
+        "x-ms-trigger": "single",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "x-ms-notification-url": {
+                  "type": "string",
+                  "description": "x-ms-notification-url",
+                  "x-ms-notification-url": true,
+                  "x-ms-visibility": "internal",
+                  "title": ""
```

Comment:
@worm333 please fill in empty titles. I know that this operation is internal, but empty properties are not allowed.